### DISCUSSION TITLE
Snapshotting the Data as well?

### DISCUSSION THREAD

**Discussion by chantlong** [directus/directus#16612] [Archive]
Hi there,
I have a DB for `dev` `staging` and `prod`.
The data is basically a read-only Database, so when I add some data, to `dev`, I expect the exact same data (collections, rows of data) to show up on `staging`.
The snapshot cli is pretty useful for copying the schema over, but when copying the data over, I've been exporting excel from `dev` 
 and importing it manually each time on the `staging` directus UI.
Is there an easier way to copy data over?

Thanks

**Comment by paescuj** [comment]
There are already similar threads on this topic, for example #11367.
